Buildings/Controls/SetPoints/Table.mo

```diff
@@ -27,7 +27,7 @@ protected
     annotation (Placement(transformation(extent={{-20,-10},{2,10}})));
 equation
   connect(u, tab.u[1]) annotation (Line(
-      points={{-120,0},{-70,0},{-70,0},{-22,0}},
+      points={{-120,0},{-70,0},{-70,0},{-22.2,0}},
```

Comment:
I'm having issues with dymola not updating the file if edited and saved in the dymola editor. Here is a screen shot showing the issue (see line 30 both in dymola and in the text editor). After I introduced the change in the text editor, the changes were saved and recorded by git.
![screenshot from 2017-02-27 13-29-51](https://cloud.githubusercontent.com/assets/5322943/23380891/5f11186e-fcf1-11e6-9d07-e454efb0bbf9.png)